website/client/js/controllers/partyCtrl.js

```diff
@@ -20,7 +20,8 @@ habitrpg.controller("PartyCtrl", ['$rootScope','$scope','Groups','Chat','User','
         $rootScope.party = $scope.group = $scope.newGroup = { type: 'party' };
       }
 
-      if ($state.is('options.social.party')) {
+      if ($state.is('options.social.party') && $rootScope.party) {
```

Comment:
Without this extra check, the header wasn't loading properly when refreshing the party page.